### ISSUE TITLE
Pagination component

### ISSUE THREAD

**Issue by taras-venger** [Panenco/panenco-ui#166]
Pagination should be wrapped into the 'nav' tag (alternatively use role="navigation"), add aria-current=true to the selected button, add default aria-labels to the buttons and container (e.g. ‘go to page x’, ‘go to next page’, ‘go to previous page’), add correct title/tooltip:

![image](https://user-images.githubusercontent.com/53059694/137799719-f57f31de-a23b-482b-a3a6-02d72571a39a.png)
